OpenShift/helm/nexus-iq/templates/deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nxiq-deployment
```

Comment:
this can also draw from templates (and with the labels & app elements in this file)